Sources/BrowserServicesKit/ContentBlocking/ContentBlockingRulesLookupTask.swift

```diff
@@ -24,19 +24,21 @@ extension ContentBlockerRulesManager {
 
     final class LookupRulesTask {
 
-        typealias LookupResult = (compiledRulesList: WKContentRuleList, model: ContentBlockerRulesSourceModel)
+        // This type has been overloaded multiple times, I'm collapsing them all into CompilationResult type
+        // typealias LookupResult = (compiledRulesList: WKContentRuleList, model: ContentBlockerRulesSourceModel)
 
         private let sourceManagers: [ContentBlockerRulesSourceManager]
 
-        public private(set) var result: [LookupResult]?
+        // todo: how to get around naming this compilation result when in lookuptask?
+        public private(set) var result: [CompilationResult]?
```

Comment:
I think naming is fine, just we should make it a proper top level struct (see my last comment) so the usage is consistent within tasks.